bindings/nodejs/examples/wallet/13-burn-native-token.ts

```diff
@@ -0,0 +1,101 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { getUnlockedManager } from './account-manager';
+
+// The native token id. Replace it with a TokenId that is available in the account, the foundry output which minted it,
+// also needs to be available. You can check this by running the `get_balance` example. You can mint a new native token
+// by running the `mint_native_token` example.
+// eslint-disable-next-line prefer-const
```

Comment:
Why not make it a const?

Reply:
if its const we cant check if the id is the default, because const cannot be changed :D

Reply:
But the example doesn't change it, it will just return?

Reply:
Yea but we want other people to copy/paste the example and change the tokenId right? If its const we cannot check if they changed it, if its let we can

Reply:
We decided to just use the foundry ID of the first output, right? Can we now merge? :trollface:

Reply:
But it also doesn't do that

Reply:
We can just leave it as is. I will change it anyway (already did basically)